Robinhood/Robinhood.py

```diff
@@ -165,53 +195,91 @@ def investment_profile(self):
         data = res.json()
         return data
 
-    def instruments(self, stock):
+    def instruments(self, query=None, symbol=None, instrumentid=None):
```

Comment:
though `symbol` is the right word in this case, `stock` is the uniform name in the API

Also, `instrument_id` would be my nit-pick for naming

Reply:
fwiw, previously this function was not searching for a stock, it was using the query param to search by name (e.g. company name or symbols), so I felt making it more explicit was better here.